consume_dependent_ptr_impl.h

```diff
@@ -54,6 +54,6 @@ template<typename T> inline dependent_ptr<T*> dependent_ptr<T>::operator&() cons
 
 template<typename T> inline T* dependent_ptr<T>::value() const { return ptr; }
 
-template<typename T> inline dependency dependent_ptr<T>::dependency() const { using shadowed = class dependency; return shadowed(ptr); }
+template<typename T> inline dependency dependent_ptr<T>::Dependency() const { using shadowed = class dependency; return shadowed(ptr); }
```

Comment:
Ditto here, `dep`, and you can then remove the `shadowed` thing, just `{ return dep(ptr); }`

Reply:
I added a pair of commits, one to use the CPU() macro and the other to use "dep" instead of "Dependency", and also simplifying the inline function's "return" statement.